x-pack/test/functional/apps/ml/data_frame_analytics/outlier_detection_creation.ts

```diff
@@ -336,6 +364,44 @@ export default function ({ getService }: FtrProviderContext) {
           );
         });
 
+        it('adds discover custom url to the analytics job', async () => {
+          await ml.testExecution.logTestStep('opens edit flyout for discover url');
+          await ml.dataFrameAnalyticsTable.openEditFlyout(testData.jobId);
+          await ml.dataFrameAnalyticsEdit.openEditCustomUrlsForJobTab(testData.jobId);
+
+          await ml.testExecution.logTestStep('add discover custom url for the analytics job');
+          await ml.dataFrameAnalyticsEdit.addDiscoverCustomUrl(testDiscoverCustomUrl);
+
+          await ml.dataFrameAnalyticsEdit.updateAnalyticsJob();
+        });
+
+        it('adds dashboard custom urls to the analytics job', async () => {
+          await ml.testExecution.logTestStep('opens edit flyout for dashboard url');
+          await ml.dataFrameAnalyticsTable.openEditFlyout(testData.jobId);
+          await ml.dataFrameAnalyticsEdit.openEditCustomUrlsForJobTab(testData.jobId);
+
+          await ml.testExecution.logTestStep('add dashboard custom url for the analytics job');
+          await ml.dataFrameAnalyticsEdit.addDashboardCustomUrl(testDashboardCustomUrl, {
+            index: 1,
+            url: `dashboards#/view/${testDashboardId}?_g=(filters:!(),time:(from:'$earliest$',mode:absolute,to:'$latest$'))&_a=(filters:!(),query:(language:kuery,query:'SaleType:\"$SaleType$\"'))`,
+          });
+
+          await ml.dataFrameAnalyticsEdit.updateAnalyticsJob();
+        });
+
+        it('adds other custom url type to the analytics job', async () => {
+          await ml.testExecution.logTestStep('opens edit flyout for other url');
+          await ml.dataFrameAnalyticsTable.openEditFlyout(testData.jobId);
+          await ml.dataFrameAnalyticsEdit.openEditCustomUrlsForJobTab(testData.jobId);
+
+          await ml.testExecution.logTestStep('add other type custom url for the analytics job');
+
+          await ml.testExecution.logTestStep('add other type custom url for the analytics job');
```

Comment:
This is a duplicate of the log above.

Reply:
Removed in 518fabb4bc2b59e729c93231c9c258c0cbffa1b3

Reply:
nit: 'adds'